app/models/pipeline_run_stage.rb

```diff
@@ -256,10 +256,20 @@ def db_load_host_filtering
     pr.save
   end
 
+  def output_json_name
+    pipeline_run.pipeline_branch == 'charles/hit-calling' ? PipelineRun::MULTIHIT_OUTPUT_JSON_NAME : PipelineRun::OUTPUT_JSON_NAME
+  end
+
+  def invalid_genus_call?(tcnt)
+    tcnt['genus_taxid'].to_i < TaxonLineage::INVALID_CALL_BASE_ID
+  rescue
+    false
```

Comment:
Why is this `false` and not `true`?

Reply:
Doesn't this function return `true` for invalid calls at any level, whether genus, species, or family?

Reply:
No, if the species call is invalid, the genus_taxid may still be valid (positive).

Reply:
The rescue case should never occur, but when in doubt, I'm returning false so that the entry doesn't get dropped. That way, we would at least see the offending entry on the report page and catch the bug. If we hide it, we may never know.